src/wasm-emscripten.cpp

```diff
@@ -499,15 +507,50 @@ std::string EmscriptenGlueGenerator::generateEmscriptenMetadata(
     meta << ", ";
     meta << "\"jsCallStartIndex\": " << jsCallWalker.jsCallStartIndex << ", ";
     meta << "\"jsCallFuncType\": [";
-    bool first = true;
+    commaFirst = true;
     for (std::string sig : jsCallWalker.indirectlyCallableSigs) {
-      if (!first) meta << ", ";
-      first = false;
+      meta << maybeComma();
       meta << "\"" << sig << "\"";
     }
     meta << "]";
   }
 
+  meta << ", \"declares\": [";
+  commaFirst = true;
+  for (const auto& import : wasm.imports) {
+    if (import->kind == ExternalKind::Function &&
+        // !import->name.startsWith(EM_JS_PREFIX.str) &&
```

Comment:
unnecessary comment?

Reply:
Technically yes. Should probably explicitly annotate this with a `TODO(jgravelle), uncomment when EM_JS lands`. I was thinking one of these would rebase on the other, and fix it then. Explicit is better though.